crates/core/src/live_socket/socket.rs

```diff
@@ -35,13 +36,13 @@ macro_rules! lock {
 #[cfg(not(test))]
 use std::sync::OnceLock;
 #[cfg(not(test))]
-static COOKIE_JAR: OnceLock<Arc<Jar>> = OnceLock::new();
+pub static COOKIE_JAR: OnceLock<Arc<Jar>> = OnceLock::new();
 
 // Each test runs in a separate thread and should make requests
 // as if it is an isolated session.
 #[cfg(test)]
 thread_local! {
-    static TEST_COOKIE_JAR: Arc<Jar> = Arc::default();
+    pub static TEST_COOKIE_JAR: Arc<Jar> = Arc::default();
```

Comment:
This could probably be `pub (crate)`.

Reply:
The plan for this PR
- get a working centralized version of the API while keeping the old `LiveSocket` one.
- run both side by side in as many of the tests as possible to ensure parity - then delete `LiveSocket`